qgis_color_func.py

```diff
@@ -0,0 +1,72 @@
+import os
+
+
+def conversion(table_path, output_path):

```

Comment:
Try to be more specific.  eg:

```suggestion
def convert_color_table_grass_to_qgis(table_path, output_path):
```

Reply:
Try to add at least minimalistic docstrings, see https://realpython.com/documenting-python-code/